package.json

```diff
@@ -33,13 +33,16 @@
     "eslint": "eslint --ext .js *.js lib tests"
   },
   "dependencies": {
+    "add": "^2.0.6",
     "archiver": "^5.3.1",
     "bluebird": "^3.7.2",
+    "find-yarn-workspace-root": "^2.0.0",
     "fs-extra": "^9.1.0",
     "glob": "^7.2.3",
     "is-builtin-module": "^3.2.0",
     "lodash": "^4.17.21",
-    "semver": "^7.3.8"
+    "semver": "^7.3.8",
+    "yarn": "^1.22.19"
```

Comment:
Not everyone is using yarn, why did you add it?

Reply:
same here